### ISSUE TITLE
Feature Request: Support for newer Powershell

### ISSUE THREAD

**Issue by vaporup** [cloudradar-monitoring/rport#346]
Windows Powershell (5) is only in maintenance mode. Further development is done on version 7.

- https://www.ipswitch.com/blog/end-of-the-road-for-windows-powershell
- https://docs.microsoft.com/en-us/powershell/scripting/install/PowerShell-Support-Lifecycle

Running scripts on Windows Clients currently are fix for **Windows Powershell** (5)

```
$PSVersionTable
```

```
Name                           Value                                                                                   
----                           -----                                                                                   
PSVersion                      5.1.19041.1023                                                                          
PSEdition                      Desktop                                                                                 
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
BuildVersion                   10.0.19041.1023                                                                         
CLRVersion                     4.0.30319.42000                                                                         
WSManStackVersion              3.0                                                                                     
PSRemotingProtocolVersion      2.3                                                                                     
SerializationVersion           1.1.0.1             
```

Usually Powershell 7 is parallel installed via the official MSI file from Microsoft.
Sometimes even Powershell 6.

Would be nice to be able supplying a path to different Powershell Interpreters.
Maybe they can be parsed from the registry but supplying a path in addition
would be good, since some people just unzip modern Powershell releases to some path
in the filesystem...

**Comment by breathbath** [comment]
Implemented here: https://github.com/cloudradar-monitoring/rport/pull/394

**Comment by thorstenkramm** [comment]
Implemented and released with 0.6.0.
